packages/xrpl/snippets/src/multisigning.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  multisign,
+  Client,
+  AccountSet,
+  convertStringToHex,
+  SignerListSet,
+} from '../../dist/npm'
+
+const client = new Client('wss://s.altnet.rippletest.net:51233')
+
+async function multisigning(): Promise<void> {
+  await client.connect()
+  const { wallet: wallet1 } = await client.fundWallet()
+  const { wallet: wallet2 } = await client.fundWallet()
+  const { wallet: walletMaster } = await client.fundWallet()
+  const signerListSet: SignerListSet = {
+    TransactionType: 'SignerListSet',
+    Account: walletMaster.classicAddress,
+    SignerEntries: [
+      {
+        SignerEntry: {
+          Account: wallet1.classicAddress,
+          SignerWeight: 1,
+        },
+      },
+      {
+        SignerEntry: {
+          Account: wallet2.classicAddress,
+          SignerWeight: 1,
+        },
+      },
+    ],
+    SignerQuorum: 2,
+  }
+
+  const signerListResponse = await client.submit(signerListSet, {
+    wallet: walletMaster,
+  })
+  console.log('SignerListSet constructed successfully:')
+  console.log(signerListResponse)
+
+  const accountSet: AccountSet = {
+    TransactionType: 'AccountSet',
+    Account: walletMaster.classicAddress,
+    Domain: convertStringToHex('example.com'),
+  }
+  const accountSetTx = await client.autofill(accountSet, 2)
+  console.log('Transaction set successfully:')
+  console.log(accountSetTx)
+  const { tx_blob: tx_blob1 } = wallet1.sign(accountSetTx, true)
+  const { tx_blob: tx_blob2 } = wallet2.sign(accountSetTx, true)
+  const multisignedTx = multisign([tx_blob1, tx_blob2])
+  const submitResponse = await client.submit(multisignedTx)
+
+  if (submitResponse.result.engine_result === 'tesSUCCESS') {
+    console.log('Transaction multisigned successfully:')
+    console.log(submitResponse)
+    if (submitResponse.result.tx_json.Signers) {
+      console.log(
+        `Signers contain ${submitResponse.result.tx_json.Signers.length} entries`,
+      )
+    }
+  } else {
+    console.log('There was an error multi-signing this transaction.')
+  }
```

Comment:
Make sure to add `await client.disconnect()` at the end of test alike to all other code snippets